time/src/main/java/io/spine/time/Timestamps2.java

```diff
@@ -74,48 +75,73 @@ public static boolean isLaterThan(Timestamp timestamp, Timestamp thanTime) {
      */
     public static Instant toInstant(Timestamp timestamp) {
         checkNotNull(timestamp);
-        return InstantConverter.INSTANCE.convert(timestamp);
+        return InstantConverter.INSTANCE.reverse()
+                                        .convert(timestamp);
+    }
+
+    /**
+     * Parses the string with a timestamp.
+     *
+     * <p>Unlike {@link com.google.protobuf.util.Timestamps#parse(String) its Protobuf counterpart}
+     * this method does not throw a checked exception.
+     *
+     * @throws IllegalArgumentException if the string is not of required format
```

Comment:
As long as you mention there is a required format, please mention the one in the first sentence of the document. I assume it's still ISO 8601.